SU2_CFD/src/numerics/NEMO/CNEMONumerics.cpp

```diff
@@ -77,6 +77,55 @@ CNEMONumerics::CNEMONumerics(unsigned short val_nDim, unsigned short val_nVar,
     }
 }
 
+CNEMONumerics::CNEMONumerics(unsigned short val_nDim, unsigned short val_nVar,
```

Comment:
This is a repeated constructor as the turbulence model needs some extra information.